src/tail.ts

```diff
@@ -0,0 +1,88 @@
+export function sleep(ms: number) {
+    return new Promise((resolve) => setTimeout(resolve, ms))
+}
+
+function processInput(timeStr: string) {
```

Comment:
I'd like to go over some improvements to this function on a call. I think it would get confusing trying to type out the recommendations.